lib/browser/api/module-list.js

```diff
@@ -0,0 +1,23 @@
+// Browser side modules, please sort alphabetically.
+module.exports = [
+  {name: 'app', file: 'app'},
```

Comment:
The docs linter is using the `decamelize` module to slugify the API names:

```
$ yarn global add trymodule && trymodule decamelize

> decamelize('BrowserWindow', '-')
'browser-window'

> decamelize('systemPreferences', '-')
'system-preferences'

> decamelize('app', '-')
'app'
```

This might be overkill here, but I figured I'd mention it.

You could also add a `sort` call to ensure that the list is alphabetical.